access/utils/cadence.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"errors"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/cadence/encoding/json"
+)
+
+func EncodeArgs(argValues []cadence.Value) ([][]byte, error) {
+	args := make([][]byte, len(argValues))
+	for i, arg := range argValues {
+		var err error
+		args[i], err = json.Encode(arg)
+		if err != nil {
+			return nil, errors.New("couldn't encode cadence value: " + err.Error())
```

Comment:
avoid using apostrophes in error/log messages since they can make it hard to search through the logs
 
```suggestion
			return nil, errors.New("could not encode cadence value: " + err.Error())
```